arbnode/seq_coordinator.go

```diff
@@ -617,17 +617,18 @@ func (c *SeqCoordinator) update(ctx context.Context) time.Duration {
 		log.Warn("sequencer is not synced", detailsList...)
 	}
 
+	processedMessages, err := c.streamer.GetProcessedMessageCount()
```

Comment:
not directly related but this is a good PR for this change:
instead of checking len(syncProgress) above - sync monitor now has a method called Synced() that returns bool and answers directly the question of if this node is synced